src/chapters/spring/exercises.rst

```diff
@@ -0,0 +1,38 @@
+Exercises: Bonjour Spring!
+==========================
+
+While reading the chapter, you created a
+basic Hello, World application using Spring Boot called ``hello-spring``. Open that project up
+in IntelliJ, and get ready to add some features!
+
+Modify your ``HelloController`` class to display a form on ``GET``
```

Comment:
on a GET request? ie, add an a between on and ``GET``

Reply:
Done! Thanks!